docs/connectors/behavior.mdx

```diff
@@ -1,7 +1,5 @@
 ---
 title: "Connector Behavior"
-sidebar_label: "Behavior"
-slug: "behavior"
```

Comment:
Will the slug now be `connector-behavior`? If so, do we maybe have any backlinks to this?

Reply:
No, this one will stay as it is (see filename). It will change for a couple of other documents in this directory, but I think I changed all links that point toward those.